cli/running/running.go

```diff
@@ -112,7 +112,7 @@ func Stop(ctx *context.Ctx) error {
 		return fmt.Errorf("No instances specified")
 	}
 
-	if err := processes.Stop(); err != nil {
+	if err := processes.Stop(ctx.Running.StopForced); err != nil {
```

Comment:
I think it could be whole `ctx.Running` not only single flag. In future we may introduce more flags and function will have too many arguments. Keep it in mind don't change it now.

Reply:
Yes, maybe sometimes we should keep all parameters in the options map or smth like this.

But `force` isn't an attribute of processes set, it's a `Stop` option. Just like `daemonize` for `Start` or `follow, n` options for `Log`.